src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Excavator is doing some work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The Excavator stopped working");
```

Comment:
Similarly, the message 'The Excavator stopped working' should indicate that it is an Excavator type of Machine that has stopped working to provide clarity.